src/lib/schemaValidation.ts

```diff
@@ -379,7 +379,9 @@ const SnapshotSchema: JSONSchemaType<Snapshot> = {
         name: {
             type: "string",
             minLength: 1,
-            errorMessage: "Invalid snapshot; name is mandatory and cannot be empty"
+            maxLength:255,
+            pattern: "^.*\\S.*$",
+            errorMessage: "Invalid snapshot; name is mandatory and cannot be empty and cannot be more than 255 characters"
```

Comment:
@Nick-1234531 can apply below suggestion
```suggestion
            errorMessage: "Invalid snapshot: name is mandatory, cannot be empty, and must not exceed 255 characters."
```